src/main.rs

```diff
@@ -1,15 +1,10 @@
-use std::{collections::HashMap, env, option::Option::Some};
-
-use llvm_ir::{
-    constant::Constant,
-    instruction::Instruction,
-    name, BasicBlock, Module,
-    Operand::{self, ConstantOperand, LocalOperand, MetadataOperand},
-    Terminator,
-};
+use std::{collections::HashMap, env, ffi::c_void, option::Option::Some};
+use either::Either::{Left, Right};
+use libc::{RTLD_DEFAULT, dlsym};
+use llvm_ir::{BasicBlock, Module, Operand::{self, ConstantOperand, LocalOperand, MetadataOperand}, Terminator, constant::Constant, instruction::{Call, Instruction}, name::Name::{Name, Number}};
```

Comment:
Does your editor auto format this or did you do that manually?

Reply:
Mostly editor. Does it need a change?

Reply:
The rust formatter will get rid of it again, once you format before we merge. But I think you should try and disable this, to avoid this showing up in the PRs every time.

Reply:
There should be an option in VSCode to have it automatically format with `rustfmt`: https://github.com/editor-rs/vscode-rust/blob/master/doc/format.md

I'm not sure what it's currently using though.

Reply:
I could just run cargo fmt before creating a PR.

Reply:
You should probably do that anyway. But every time you add a commit to the PR you need to run it again as your editor might reformat things again. So it might be worthwhile to check how to get VSCode to apply the correct formatting in the first place.